.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
It would be nice to move the grunt invocation to an npm script like `npm run dist` or `npm run ci` since then we can remove the `npm install -g` calls. npm scripts automatically add `node_modules/.bin/` to the PATH before running.

I noticed some of the raincatcher modules have `sudo: true` but I'm not sure why, because since it runs on `nvm` it doesn't need root permissions to install global modules anyway, it's worth checking if the builds really need that.